### ISSUE TITLE
GSConnect never updates the device volume to MPRIS

### ISSUE THREAD

**Issue by rohmishra** [andyholmes/gnome-shell-extension-gsconnect#888]
**Describe the bug**

When creating MPRIS  Player GSConnect sets the default volume to 1.0 but never updates it when volume on device is changed. GSConnect logs indicate that GSConnect does get the volume information from the device when it receives song info..
I also found that it is used as default when it fails to get the default value. https://github.com/andyholmes/gnome-shell-extension-gsconnect/blob/c2b12f767731c4f4665b30df67a1550f29e49ca5/src/service/components/mpris.js#L429

Changing the volume works as intended and correctly updates the volume

**Steps To Reproduce:**

1. Play music on phone
2. Once GSConnect creates the MPRIS notifcation, check volume from MPRIS.

**Expected behavior**
GSConnect reports the correct value (divided by 100 for value between 0 and 1)

**Screenshots**

![MPRIS Data](https://user-images.githubusercontent.com/2221064/85627575-b473c380-b63c-11ea-97fa-aa32b450aed8.png)

**Support Log**

Please generate a support log ([Instructions](../wiki/Help#generate-support-log)) and paste any messages related to this issue between the two ``` lines below.

```

Jun 30 12:01:04 org.gnome.Shell.Extensions.GSConnect[9733]: [/service/protocol/core.js:receive/<:298]: winterbell: {
                                                              "id": 1593532865426,
                                                              "type": "kdeconnect.mpris",
                                                              "body": {
                                                                "player": "YouTube",
                                                                "nowPlaying": "LastWeekTonight - Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "title": "Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "artist": "LastWeekTonight",
                                                                "album": "",
                                                                "isPlaying": false,
                                                                "pos": 859614,
                                                                "length": 1271000,
                                                                "canPlay": true,
                                                                "canPause": true,
                                                                "canGoPrevious": false,
                                                                "canGoNext": true,
                                                                "canSeek": true,
                                                                "volume": 48
                                                              }
                                                            }
Jun 30 12:01:04 org.gnome.Shell.Extensions.GSConnect[9733]: [/service/protocol/core.js:receive/<:298]: winterbell: {
                                                              "id": 1593532865433,
                                                              "type": "kdeconnect.mpris",
                                                              "body": {
                                                                "player": "YouTube",
                                                                "nowPlaying": "LastWeekTonight - Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "title": "Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "artist": "LastWeekTonight",
                                                                "album": "",
                                                                "isPlaying": false,
                                                                "pos": 859614,
                                                                "length": 1271000,
                                                                "canPlay": true,
                                                                "canPause": true,
                                                                "canGoPrevious": false,
                                                                "canGoNext": true,
                                                                "canSeek": true,
                                                                "volume": 48
                                                              }
                                                            }
Jun 30 12:01:04 org.gnome.Shell.Extensions.GSConnect[9733]: [/service/protocol/core.js:receive/<:298]: winterbell: {
                                                              "id": 1593532865442,
                                                              "type": "kdeconnect.mpris",
                                                              "body": {
                                                                "player": "YouTube",
                                                                "nowPlaying": "LastWeekTonight - Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "title": "Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "artist": "LastWeekTonight",
                                                                "album": "",
                                                                "isPlaying": false,
                                                                "pos": 859614,
                                                                "length": 1271000,
                                                                "canPlay": true,
                                                                "canPause": true,
                                                                "canGoPrevious": false,
                                                                "canGoNext": true,
                                                                "canSeek": true,
                                                                "volume": 48
                                                              }
                                                            }
Jun 30 12:01:04 org.gnome.Shell.Extensions.GSConnect[9733]: [/service/protocol/core.js:receive/<:298]: winterbell: {
                                                              "id": 1593532865447,
                                                              "type": "kdeconnect.mpris",
                                                              "body": {
                                                                "player": "YouTube",
                                                                "nowPlaying": "LastWeekTonight - Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "title": "Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "artist": "LastWeekTonight",
                                                                "album": "",
                                                                "isPlaying": false,
                                                                "pos": 859614,
                                                                "length": 1271000,
                                                                "canPlay": true,
                                                                "canPause": true,
                                                                "canGoPrevious": false,
                                                                "canGoNext": true,
                                                                "canSeek": true,
                                                                "volume": 48
                                                              }
                                                            }
Jun 30 12:01:04 org.gnome.Shell.Extensions.GSConnect[9733]: [/service/protocol/core.js:receive/<:298]: winterbell: {
                                                              "id": 1593532865451,
                                                              "type": "kdeconnect.mpris",
                                                              "body": {
                                                                "player": "YouTube",
                                                                "nowPlaying": "LastWeekTonight - Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "title": "Facial Recognition: Last Week Tonight with John Oliver (HBO)",
                                                                "artist": "LastWeekTonight",
                                                                "album": "",
                                                                "isPlaying": false,
                                                                "pos": 859614,
                                                                "length": 1271000,
                                                                "canPlay": true,
                                                                "canPause": true,
                                                                "canGoPrevious": false,
                                                                "canGoNext": true,
                                                                "canSeek": true,
                                                                "volume": 48
                                                              }
                                                            }
```

**System Details (please complete the following information):**

 - **GSConnect version:** [39]
 - **Installed from:** [GNOME Extensions Website]
 - **GNOME/Shell version:** [3.36.2]
 - **Distro/Release:** [Manjaro]

**GSConnect environment (if applicable):**

 - **Paired Device(s):** [ Pixel 3 ]
 - **KDE Connect Version:** [1.14.2]

**Comment by rohmishra** [comment]
https://github.com/JasonLG1979/gnome-shell-extension-mpris-indicator-button/issues/41 

if using with MPRIS indicator, this causes the volume to increase to maximum under some conditions because of a workaround employed by said extension.

**Comment by andyholmes** [comment]
Thanks for reporting with thorough logs. Should be fixed now in 80a19a8, just a litte typo :)
